runtime/meta/ffi.cpp

```diff
@@ -258,10 +266,22 @@ extern "C" {
     return ffiCall(true, addr, args, fixtypes, vartypes, ret);
   }
 
+
   SortInt hook_FFI_address(SortString fn) {
     char * func = getTerminatedString(fn);
-    void * handle = so_lib_handle();
-    void * address = dlsym(handle, func);
+
+    std::string funcStr = func;
+    std::map<std::string, void *> privateSymbols;
```

Comment:
Not sure it's worth changing, but this could be a const static variable. Of course, the `const` would mean that the `if` below should use iterators.

Reply:
I agree it could be, but I couldn't figure out an easy way to initialize a std::map with a specific set of elements, and thus went with this approach. In the unlikely event that this method reports a lot of self time, we can optimize.

Reply:
Wouldn't this work?
```
std::map<std::string, void *> privateSymbols = {
    {"atexit", (void *)atexit},
    {"at_quick_exit", (void *)at_quick_exit}
};
```